walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -20,9 +20,11 @@
 import java.util.Arrays;
 
 public final class FindMeetingQuery {
-    Collection<TimeRange> availibility;
+  Collection<TimeRange> availibility;
```

Comment:
When I tried switching this to inside the next set of {} I kept getting an error saying availability had not been initialized...